src/OrbitCaptureClient/CaptureClient.cpp

```diff
@@ -95,6 +96,34 @@ Future<ErrorMessageOr<CaptureListener::CaptureOutcome>> CaptureClient::Capture(
   return capture_result;
 }
 
+[[nodiscard]] static std::vector<ApiFunction> FindApiFunctions(
+    const orbit_client_data::ModuleManager& module_manager) {
+  // Api functions are declared in Orbit.h. They are implemented in user code through the
+  // ORBIT_API_INSTANTIATE macro. Those functions are used to query the tracee for Orbit specific
+  // information such as the memory location where Orbit should write function pointers to enable
```

Comment:
```suggestion
  // information, such as the memory location where Orbit should write function pointers to enable
```
This is a long sentence.

Reply:
Very good, thanks. Done.